app-web/topicRegistry/community-contributed.json

```diff
@@ -136,6 +136,18 @@
             ]
         }
     },
+    {
+      "sourceType":"github",
+      "resourceType": "Documentation",
+      "sourceProperties":{
+      "url":"https://github.com/bcgov/ExchangeLabOps/tree/master/Resident-Teams",
+      "repo":"ExchangeLabOps",
+      "owner":"bcgov",
+      "files": [
+        "README.md"
```

Comment:
this should be the change to the path based on the base url: "Resident-Teams/README.md"